TestProject/output/ConsoleOutput.php

```diff
@@ -5,13 +5,24 @@
 use options\Getopt;
 use cellularAutomat\Board;
 
+/**
+ * Shows the output of the current board over the console.
+ *
+ * @class ConsoleOutput
+ */
 class ConsoleOutput extends BaseOutput
 {
+
+    /**
+     * Shows the output of the current board over the console.
+     *
+     * @param Board $_board Presents the current board.
+     */
     function outputBoard(Board $_board)
     {
         for ($y = 0; $y < $_board->getHeight(); ++$y) {
             for ($x = 0; $x < $_board->getWidth(); ++$x) {
-                $_board->setCell($x, $y, rand(0, 1));
+                $_board->printOutBoard();
```

Comment:
You now print the board x*y times for every generation!
The purpose of this phase of the development is, to seperate the code better. So since we now have a console output, the board should not have any printOutBoard() method anymore. So please remove this method and implement the console output in this class completely. So the loop you added is actually correct, but you must only output once cell in it.